.github/workflows/integration.yml

```
@@ -33,6 +33,8 @@ jobs:
         include:
           - TEST_NAME: "MariaDB 11.4"
             ARGS: MYSQL_VER=mariadb:11.4.3
+          - TEST_NAME: "Opensearch 3.2.0"
+            ARGS: ES_VER="opensearchproject/opensearch:3.2.0"
```

Do not add a new test, just reuse an existing one:

```suggestion
- TEST_NAME: "MariaDB 11.4, OS 3.2"
ARGS: MYSQL_VER=mariadb:11.4.3 ES_VER="opensearchproject/opensearch:3.2.0"
```